bittensor_cli/src/bittensor/utils.py

```diff
@@ -558,7 +558,10 @@ def format_error_message(error_message: Union[dict, Exception]) -> str:
             err_type = error_message.get("type", err_type)
             err_name = error_message.get("name", err_name)
             err_docs = error_message.get("docs", [err_description])
-            err_description = err_docs[0] if err_docs else err_description
+            if isinstance(err_docs, list):
+                err_description = "; ".join(err_docs)
```

Comment:
```suggestion
                err_description = " ".join(err_docs)
```
Pls use just `" "` instead of `"; "` bc:

- this is the real docs of `NonAssociatedColdKey` error.
```py
['Request to stake, unstake or subscribe is made by a coldkey that is not associated with', 'the hotkey account.']
```
- this is the real docs of `NotEnoughBalanceToStake` error.
```py
['The caller is requesting adding more stake than there exists in the coldkey account.', 'See: "[add_stake()]"']
```